agent/pkg/api/v1/error.go

```diff
@@ -1,4 +1,4 @@
-package model
```

Comment:
Is this good place for errors in API pkg @nandor-magyar ?

Reply:
These errors are used by the REST interface, that's why I moved them to the v1 model. 🤔

Reply:
Yes, I like it, there is no better place for it right now.